applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -430,31 +431,31 @@ static int bis_headset_cleanup(bool from_sync_lost_cb)
 
 static int change_active_audio_stream(void)
 {
-	int ret = 0;
+	int ret;
 
-	playing_state = false;
+	if (broadcast_sink != NULL) {
```

Comment:
We tend to do "if (broadcast_sink == NULL) {return;}" in the start of the function.
It is often easier to read that way, instead of having this "if" that covers several lines of code, and then an "else"

Reply:
Agreed, like this:
```suggestion
	if (broadcast_sink == NULL) {
	    LOG_WRN("No streams");
	    return -ECANCELED;
	}
```
And then everything below is not within an if/else